client/task_update.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package client
+
+import (
+	"context"
+
+	"github.com/dragonflyoss/Dragonfly/apis/types"
+)
+
+// TaskUpdate update a task in supernode.
+func (client *APIClient) TaskUpdate(ctx context.Context, id string, config *types.TaskUpdateRequest) error {
+	resp, err := client.put(ctx, "/tasks"+id, nil, config, nil)
```

Comment:
I think the url path should be `"/tasks/"+id`. You have missed one `/`.

Reply:
sorry,updated